Create toBinary.cpp

firatt opened the pull request.

firatt committed:
Create toBinary.cpp

firatt commented:
deneme 1

atcemgil reviewed (commented):
don't use printf, use cout instead

firatt committed:
updated output format

input can be both negative and positive now. also, the output is formatted to 4 x 8 bits format for the ease of reading.
